### PR TITLE
Validation & Prettier

### PR DESCRIPTION
**Pull request by Trikolon**
This PR adds local scripts for JSON schema validation and reformatting code via Prettier.

The rule JSON schema is fetched remotely from https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/cookiebanners/schema/CookieBannerRule.schema.json

Setup via
```
npm install
```

Validate rule list JSON:
```
npm run validate
```

Check formatting with prettier:
```
npm run prettier
```

### CODE REVIEW

**Comment by Trikolon** [comment]
I've also added a GitHub action which runs these scripts.

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./CookieBannerRuleList.schema.json",
```
Comment (artines1) [inline cookie-banner-rules-list.json:2]: One naive question here though, why do we need to add `$schema` here?

**Review by artines1 — APPROVED** [review]
Thanks for working on this. I don't see any particular issue here. Just have one simple question.

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Trikolon) [reply cookie-banner-rules-list.json:2]: That's for editors to know which schema to validate the file against. E.g. vscode has some tooling for this. It's not used by the validation script.

**Pull request merged**